src/main/java/com/conveyal/taui/controllers/AggregationAreaController.java

```diff
@@ -56,7 +60,16 @@
             .build();
     private static final FileItemFactory fileItemFactory = new DiskFileItemFactory();
 
-    public static AggregationArea createAggregationArea (Request req, Response res) throws Exception {
+    /**
+     * Create binary .grid files for aggregation (aka mask) areas, save them to S3, and persist their details.
+     * @param req Must include a shapefile on which the aggregation area(s) will be based. If request includes
+     *            "union=true", features will be merged to a single aggregation area with the provided name. If not,
+     *            each feature will be its own aggregation area, named with its value of the attribute specified by
+     *            the provided name.
+     */
+
+    public static List<AggregationArea> createAggregationArea (Request req, Response res) throws Exception {
```

Comment:
If the most common mode of operation is creating N aggregation areas (one for each feature) then the method name should be plural.